Generators/AutoEquality.cs

```diff
@@ -219,10 +219,18 @@ List<MemberInfo> GetMemberInfo()
                     switch (symbol)
                     {
                         case IFieldSymbol { Type: { }, IsImplicitlyDeclared: false } fieldSymbol:
-                            list.Add(new MemberInfo(fieldSymbol.Name, fieldSymbol.Type.ToDisplayString(SymbolDisplayFormat.FullyQualifiedFormat), UseOperator(fieldSymbol.Type)));
+                            list.Add(new(
+                                fieldSymbol.Name, 
+                                fieldSymbol.Type.ToDisplayString(SymbolDisplayFormat.FullyQualifiedFormat),
+                                UseOperator(fieldSymbol.Type),
+                                IsString(fieldSymbol.Type)));
                             break;
                         case IPropertySymbol { IsIndexer: false, GetMethod: { } } propertySymbol:
-                            list.Add(new MemberInfo(propertySymbol.Name, propertySymbol.Type.ToDisplayString(SymbolDisplayFormat.FullyQualifiedFormat), UseOperator(propertySymbol.Type)));
+                            list.Add(new(
```

Comment:
Did you add a unit test :wink:

Reply:
Thanks for keeping me honest, I just added several new tests - in addition to the ones that I've updated and added from before. It should be good to go now.